driver-core/src/main/com/mongodb/client/model/UpdateOptions.java

```diff
@@ -256,6 +258,32 @@ public UpdateOptions let(final Bson variables) {
         return this;
     }
 
+    /**
+     * Gets the sort criteria to apply to the query. The default is null, which means that the documents will be returned in an undefined
+     * order.
+     *
+     * @return a document describing the sort criteria
+     * @since 5.3
+     * @mongodb.driver.manual reference/method/cursor.sort/ Sort
```

Comment:
Should we remove the manual reference here and from the `getSort` method, given that it seems more relevant to the cursor?

Reply:
I referenced the `findOptions` and the other `findOneAnd..Options` and thought it might be helpful to have a reference! I don't have a strong opinion here though.